### ISSUE TITLE
Automatic splitting breaks due to invalid Condor submission files

### ISSUE THREAD

**Issue by matz-e** [dmwm/CRABServer#5447]
```
03/22/17 14:27:38 From submit: Submitting job(s)
03/22/17 14:27:38 From submit: ERROR: Parse error in expression:
03/22/17 14:27:38 From submit:  CRAB_EDMOutputFiles = ['ttl_test.root', 'tll_test.root']
03/22/17 14:27:38 From submit:  ^^^
03/22/17 14:27:38 From submit: Error in submit file
03/22/17 14:27:38 failed while reading from pipe.
03/22/17 14:27:38 Read so far: Submitting job(s)ERROR: Parse error in expression:       CRAB_EDMOutputFiles = ['ttl_test.root', 'tll_test.root']
+^^^Error in submit file
03/22/17 14:27:38 ERROR: submit attempt failed
03/22/17 14:27:38 submit command was: /usr/bin/condor_submit -a dag_node_name' '=' 'Job0SubJobs -a +DAGManJobId' '=' '6704195 -a DAGManJobId' '='
+'6704195 -a submit_event_notes' '=' 'DAG' 'Node:' 'Job0SubJobs -a dagman_log' '='
+'/data/srv/glidecondor/condor_local/spool/4195/0/cluster6704195.proc0.subproc0/RunJobs.dag.nodes.log -a +DAGManNodesMask' '='
+'"0,1,2,4,5,7,9,10,11,12,13,16,17,24,27" -a DAG_STATUS' '=' '0 -a FAILED_COUNT' '=' '0 -a +DAGParentNodeNames' '=' '"Job0" -a notification' '='
+'never RunJobs0.subdag.condor.sub
03/22/17 14:27:38 Job submit failed after 6 tries. 
```

Code at fault identified, will try to fix ASAP.

**Comment by matz-e** [comment]
To note, I'm not sure how this passed through testing?  I just revived my dev VM from the dead, and my code that I used to test with is the same that seems to have triggered this.

**Comment by PerilousApricot** [comment]
CI testing is best testing :)

**Comment by matz-e** [comment]
Well, still need to hook up a test for it, though. This is the same code that I used to produce the plots for CHEP, that's what's throwing me off…

**Comment by belforte** [comment]
@matz-e I would have thought this to be be fatal and therefore that you put in a fix by now, since things work.  In case, can you add a reference to the PR and close this ?

**Comment by matz-e** [comment]
Fixed with #5468. This was only triggered for automatic splitting, and had been deployed on preprod early.
